frontend/src/metabase-types/api/user.ts

```diff
@@ -12,4 +12,5 @@ export interface User {
   last_login: string;
   has_invited_second_user: boolean;
   personal_collection_id: number;
+  can_access_data_model?: boolean;
```

Comment:
Is it really optional? I think that the backend will always return this field. Could you make it required and set to false in the mock creator function?